### PR TITLE
User update info

### PR DESCRIPTION
**Pull request by harrisonmuskat**
- Added feature so user can edit account and password
- Added styling to app and user profile page
- Tests pass

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -7,7 +7,6 @@ class User < ApplicationRecord
   validates :username, presence: true
   validates :first_name, presence: true
   validates :last_name, presence: true
-  validates :email, presence: true
```
Comment (enthusiastick) [inline app/models/user.rb]: Encapsulated by the uniqueness validation below. Nice catch!

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -13,16 +13,30 @@
...
+            </ul>
+          </div>
+        </div>
+    </div>
     <p class="notice"><%= notice %></p>
     <p class="alert"><%= alert %></p>
-
     <%= yield %>
 
   </body>
```
Comment (enthusiastick) [inline app/views/layouts/application.html.erb]: Nice layout.

**Comment by harrisonmuskat** [comment]
[As an authenticated user I want to update my information So that I can keep my profile up to date](https://trello.com/c/lJE7eKtX/22-as-an-authenticated-user-i-want-to-update-my-information-so-that-i-can-keep-my-profile-up-to-date)

File: config/routes.rb [hunk 3]
```diff
@@ -1,8 +1,6 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: "shows#index"
-  resources :shows do
-    resources :users
-  end
-
+  resources :shows
+  resources :users
```
Comment (enthusiastick) [inline config/routes.rb]: I would ideally like to see these scoped down to only those routes you actually intend on utilizing.

File: spec/features/04_user_updates_account_spec.rb [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    click_link "Sign in"
+
+    fill_in("Username", with: "jdoe")
+    fill_in("Password", with: "password123")
+
+    click_button "Log in"
+
+    expect(page).to have_content("Signed in successfully.")
+  end
+end
```
Comment (enthusiastick) [inline spec/features/04_user_updates_account_spec.rb]: What a lovely test.

**Pull request merged**